src/lib/app.js

```diff
@@ -78,15 +77,16 @@ class App {
       models.Transfer.createBodyParser(),
       transfers.putResource)
 
-    // TODO Need to add body parser for fulfillment object: https://github.com/interledger/five-bells-shared/blob/master/schemas/ConditionFulfillment.json#L32
     router.put('/transfers/:id/fulfillment',
       passport.authenticate(['basic', 'http-signature', 'anonymous'], {
         session: false
-      }), function * (next) {
-        this.body = yield parseBody(this)
-        yield next
-      },
+      }), models.ConditionFulfillment.createBodyParser(),
       transfers.putFulfillment)
+    router.get('/transfers/:id/fulfillment',
+      passport.authenticate(['basic', 'http-signature', 'anonymous'], {
```

Comment:
I think we should only allow "anonymous" mode when necessary. It's not necessary here is it? What about for putting a fulfillment?